RPerformanceTracking/Private/_RPTClassManipulator.m

```diff
@@ -185,4 +190,9 @@ + (NSString *)_keyForSelector:(SEL)selector class:(Class)classObj
     return (selector && classObj) ? [NSString stringWithFormat:@"%@-%@", NSStringFromSelector(selector), NSStringFromClass(classObj)] : nil;
 }
 
++ (BOOL)isSwizzleMapAvailable
```

Comment:
Does class load order affect what will be returned from this method? What if classes which require swizzling would be loaded before this class?

Reply:
We will disable swizzling for upcoming lauch becasue the swizzlings have been already done before the app updates the config.
We want to disable the swizzling by setting the activation ration to 0 on the portal, but if the activation ration is set to 0 on the portal, the server won't send anything, so the config will be nil. So we will disable the method swizzling if the config is nil.

Reply:
I still see this PR as a wrong approach of problems of perftrack crashing specific app. Instead of disabling swizzling on next app launch (still having crash on current one) we might consider checking the root of the crash. Plus tracking will be disabled only if we would be lucky enough to download the config, which might also be not a case.